### PR TITLE
RU: Translation for tutorials/load_data/images.ipynb

### PR DESCRIPTION
**Pull request by stabuev**

### CODE REVIEW

File: site/ru/tutorials/load_data/images.ipynb [hunk 1]
```diff
@@ -0,0 +1,1654 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vkM-IpB-6URx"
+      },
+      "source": [
+        "### Просмотрите изображения\n",
+        "Сейчас давайте быстро просмотрим парочку иображений, чтобы вы знали с чем имеете дело:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "изображений"

I think this should sound somewhat better, take a look:
"Для ознакомления с форматом данных выведем на экран несколько изображений из датасета:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 2]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "YgvrWLKG67-x"
+      },
+      "source": [
+        "Используя тот же метов `from_tensor_slices` вы можете собрать датасет меток:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Используя тот же метод `from_tensor_slices`, вы сможете собрать датасет из меток:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 3]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "jHjgrEeTxyYz"
+      },
+      "source": [
+        "Поскольку датасеты в одном и том же порядке вы можете просто связать (zip) их вместе чтобы получить датасет пар `(image, label)`:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: Попробовал улучшить предложение уточнениями, возможно так будет лучше:

"Поскольку датасеты следуют в одном и том же порядке, вы можете просто собрать их вместе при помощи функции zip в набор данных пары `(image, label)`:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 4]
```diff
@@ -0,0 +1,1654 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "6JsM-xHiFCuW"
+      },
+      "source": [
+        "Здесь необходимо отметить несколько вещей:\n",
+        "\n",
+        "1. Порядок важен.\n",
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "1. Важна последовательность.\n",

File: site/ru/tutorials/load_data/images.ipynb [hunk 5]
```diff
@@ -0,0 +1,1654 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "f_glpYZ-nYC_"
+      },
+      "source": [
+        "Вы готовы обучать модель.\n",
+        "\n",
+        "Отметим что для демонстрационных целей вы запустите только 3 шага на эпоху, но обычно вам нужно указать действительное число шагов, как определено ниже, перед передачей их в `model.fit()`:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: Запятая после отметим

File: site/ru/tutorials/load_data/images.ipynb [hunk 6]
```diff
@@ -0,0 +1,1654 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "UMVnoBcG_NlQ"
+      },
+      "source": [
+        "## Производительность\n",
+        "\n",
+        "Примечание: Этот раздел лишь показывает пару простых приемов которые могут помочь производительности. Для более глубокого изучения см. [Производительность входного конвейера](https://www.tensorflow.org/guide/performance/datasets).\n",
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: Запятая после приемов

File: site/ru/tutorials/load_data/images.ipynb [hunk 7]
```diff
@@ -0,0 +1,1654 @@
...
+        "\n",
+        "Простые конвейеры использованные выше читают каждый файл отдельно во время каждой эпохи. Это нормально для локального обучения на CPU, может быть недостаточно для обучения на GPU и абсолютно неприемлемо для любого вида распределенного обучения."
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Простые конвейеры, использованные выше, прочитывают каждый файл отдельно во время каждой эпохи. Это подходит для локального обучения на CPU, может быть недостаточно для обучения на GPU и абсолютно неприемлемо для любого вида распределенного обучения."

File: site/ru/tutorials/load_data/images.ipynb [hunk 8]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "lV1NOn2zE2lR"
+      },
+      "source": [
+        "Используйте `tf.data.Dataset.cache` чтобы с легкостью кешировать вычисления от эпохи к эпохе. Это очень эффективно особенно если данные помещаются в память.\n",
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Используйте `tf.data.Dataset.cache`, чтобы с легкостью кэшировать вычисления от эпохи к эпохе. Это особенно эффективно если данные помещаются в память.\n"

File: site/ru/tutorials/load_data/images.ipynb [hunk 9]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "oNmQqgGhLWie"
+      },
+      "source": [
+        "Чтобы исследовать производительность наших датасетов сперва постройте простую функцию::"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Чтобы исследовать производительность наших датасетов, сперва постройте простую функцию:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 10]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "qqo3dyB0Z4t2"
+      },
+      "source": [
+        "У файла кеша есть также то преимущество, что оно может быть использовано для быстрого перезапуска датасета без перестраивания кеша. Заметьте наскольо но быстрее во второй раз:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Также у файла кеша есть преимущество использования быстрого перезапуска датасета без перестраивания кеша. Обратите внимание, насколько быстрее это происходит во второй раз:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 11]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "flR2GXWFKcO1"
+      },
+      "source": [
+        "Затем построим датасет который читает из TFRecord файла и декодирует/переформатирует изображения с использованием функции `preprocess_image` которую вы задали ранее:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Затем построим датасет, который прочитывает файл TFRecord и обрабатывает изображения с использованием функции `preprocess_image`, которую вы задали ранее:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 12]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "cRp1eZDRKzyN"
+      },
+      "source": [
+        "Сшейте этот датасет с датасетом меток который вы определили ранее чтобы получить ожидаемый `(image,label)` pairs:"
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Объедините этот датасет с датасетом меток, который вы определили ранее, чтобы получить пару из `(image,label)`:"

File: site/ru/tutorials/load_data/images.ipynb [hunk 13]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "wb7VyoKNOMms"
+      },
+      "source": [
+        "Это медленнее `cache` версии поскольку вы не кешируете препроцессинг."
```
Comment (0101011) [inline site/ru/tutorials/load_data/images.ipynb]: "Это медленнее `cache` версии, поскольку обработанные изображения не кешируются."

File: site/ru/tutorials/load_data/images.ipynb (hunk 1 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 2 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 3 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb [hunk 14]
```diff
@@ -0,0 +1,1654 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "wwZavzgsIytz"
+      },
+      "source": [
+        "Чтобы обучить модель с этим датасетом вы захотите чтобы данные:\n",
```
[the remark this answers is not in the document]
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 4 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 6 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 7 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 8 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 9 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 10 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 11 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 12 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

File: site/ru/tutorials/load_data/images.ipynb (hunk 13 above)
Comment (stabuev) [reply site/ru/tutorials/load_data/images.ipynb]: fixed

**Comment by stabuev** [comment]
Fixed all review notes. @0101011 thank you for valuable review.
